cfg/17.06/definitions.yaml

```diff
@@ -857,7 +897,8 @@ groups:
   checks:
   - id: 4.1
     description: "Ensure a user for the container has been created (Scored)"
-    audit: "docker ps --quiet --all | xargs docker inspect --format '{{ .Id }}: User={{ .Config.User }}'"
+    failed_description: "Show this if failed"
+    audit: "docker ps --quiet --all | xargs docker inspect --format 'Id$${{ .Id }},Name$${{ .Name }}: User={{ .Config.User }}'"
```

Comment:
I'd rather we didn't change these audit commands unless we absolutely have to, as they come straight from the CIS spec, and we're supposed to be implementing that spec. I think these changes are overthinking what a user needs to see in order to make sense of the output, as the CIS spec tells them what they are looking for. For example in this case "The above command should return container username or user ID. If it is blank it means, the container is running as root."  (Well, arguably the spec could clarify that it's per container, but hopefully you see what I mean). 

Arguably we could decide to do more in the Aqua product to help here but we could parse the output and flag up the specific containers that are problematic (with blank username / ID), rather than showing the list of all containers which this will currently output. 

So, I'm not convinced that adding labelling is adding much, but it's creating a small wedge between us and the contents of the CIS spec, so I'd rather we didn't do that. Are there any cases where the changes to the audit command does more than add labelling?